java-frontend/src/test/java/org/sonar/java/se/JavaCheckVerifier.java

```diff
@@ -207,6 +205,7 @@ private void assertMultipleIssue(Set<AnalyzerMessage> issues) throws AssertionEr
     Preconditions.checkState(!issues.isEmpty(), "At least one issue expected");
     List<Integer> unexpectedLines = Lists.newLinkedList();
     Multimap<Integer, Expectations.Issue> expected = expectations.issues;
+    expectations.reverseFlows(); // FIXME temporary workaround
```

Comment:
Do you have a ticket for that? If not, create one and add a reference in the code. So we will not forget about it.